docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+    * [Launching a Linux Virtual Machine (VM)](https://aws.amazon.com/getting-started/tutorials/launch-a-virtual-machine/?trk=gs_card)
+
+3. Download and install score-client software on the AWS VM. Score-client download instructions [here](https://docs.icgc.org/download/guide/#installation-of-the-score-client)
+
+4. Next, you will need to obtain your access token to download data from AWS. Once your DACO application has been approved, use your OpenID (ie. the Gmail email address you specified in your DACO application) to log into [https://dcc.icgc.org/](https://dcc.icgc.org)
+
+    ![Portal-Login](images/Portal_login.png)
+
+5. After successful authentication, you will know that you have Cloud Access to the controlled tier if the "Login link is replaced with a green cloud icon
+
+    ![DACO-Cloud-Access](images/daco-cloud-access.png)
+
+6. Click on Token Manager
+    ![Token-Manager-Link](images/token-manager-link.png)
+
+7. Select `aws.download` and click "Generate" button to create an access token for downloading data from Collaboratory
+
+    ![Token-Manager](images/aws-token-manager-screenshot.png)
+
+
+8\. Copy and paste this token into your config file for score-client *on the AWS VM*. Click [here](https://docs.icgc.org/download/guide/#access-configuration) for instructions on how to configure other elements of score-client. The configuration of the Score Client is stored in the `conf/application.properties` file of the distribution. Edit `application.properties` file to add the generated acesss token.
+
+    Example manifest user configuration
+    ```
+    #
+    # Defines the ICGC access token for authorized access to data
+    # 
+    accessToken=paste_your_aws_token_here
+    
+    #
+    # Controls the number of concurrent threads for multi-part data transfers. It is recommended to set this to the number of cores of the Compute Instance.
+    #
+    transport.parallel=6
+    
+    #
+    # Controls the amount of non-heap memory per thread, in gigabytes. It is recommended set this to a value of 1 (1 GB). Be sure to leave enough memory for the operating system and any other software that may be running on the Compute Instance
+    #
+    transport.memory=1
+    ```
+
+#### Download Data Instructions
+1. Find your data of interest in the [Data Repository](https://dcc.icgc.org/repositories) by selecting `AWS` repository. Click on "Manifest ID"
+    ![Generate-AWS-Manifest](images/generate-aws-manifest.png)
+
+2. This will generate a score-client command which you can use to download files from AWS
+    ![Score-client-command](images/score-client-command.png)
+
+3. Make sure you are logged into your AWS VM. Once logged in, run the score-client command *on the AWS VM* (Remember to specify the download directory using the `--output-dir` flag)
+
+```
+bin/score-client download --manifest 19fa5fe2-ce2e-4657-8072-8c7a30e70847 --output-dir aws_downloads
```

Comment:
consider adding `--profile aws` in the command example, although the default is already `aws`, I think it's better to be explicit.

Reply:
Actually, another question here, why use `Manifest ID` in this example? Why not use `Download Manifest`, same as what used in Collab download?

In general, I think we should avoid using `Manifest ID` as the way to download, it could really become a black box, users will not know how many files to be downloaded. Also hard to resume should downloading large number of files failed in the middle.

Reply:
I updated the code to use `--profile aws` and removed using `Manifest ID`. Updated instructions to use downloaded manifest file instead.